ci/docker-run.sh

```diff
@@ -76,6 +76,19 @@ ARGS+=(
   --env CRATES_IO_TOKEN
 )
 
+if [[ $BUILDKITE_LABEL == "checks" ]] && echo "$BUILDKITE_BRANCH" | grep -E '^pull/[0-9]+/head$'; then
+  api_base="https://api.github.com/repos/solana-labs/solana/pulls"
+  pr_num=$(echo "$BUILDKITE_BRANCH" | grep -Eo '[0-9]+')
+  branch=$(curl -s "$api_base/$pr_num" | ruby -r json -e 'puts JSON.parse(STDIN.read())["head"]["ref"]')
```

Comment:
I don't recall a Ruby dependency anywhere else, so would prefer not to add it here. Can you do this with Python?

Reply:
sure thing! Moreover, I wanted to ask we can install `jq` someway? If not, I'll go to the `python` path. :)

Reply:
As tiny as jq is, I'd prefer we stick to dependencies that are available by default on Ubuntu and MacOS. If that restriction isn't practical, add jq to the Dockerfile.

Reply:
I'm ok with requiring `jq` (but yes it would need to be added to `ci/docker-rust/Dockerfile`), but `python` would be just fine too.

Reply:
Ok, I just switched to `python`. :)